fdbcli/TenantCommands.actor.cpp

```diff
@@ -686,14 +776,15 @@ Future<bool> tenantCommandForwarder(Reference<IDatabase> db, std::vector<StringR
 	}
 
 	return tenantCommand(db, forwardedTokens);
-} // namespace fdb_cli
+}
 
 void tenantGenerator(const char* text,
                      const char* line,
                      std::vector<std::string>& lc,
                      std::vector<StringRef> const& tokens) {
 	if (tokens.size() == 1) {
-		const char* opts[] = { "create", "delete", "deleteId", "list", "get", "configure", "rename", nullptr };
+		const char* opts[] = { "create",    "delete", "deleteId", "list",   "get",
+			                   "configure", "rename", "lock",     "unlock", nullptr };
```

Comment:
We should add a generator for the "lock" command as well similar to the other commands in this if/else sequence for the r and rw options.

Reply:
Also, it would be helpful to add a hint generator for lock in the next function